<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1061</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1061</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Reconstruction by Hanming</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="openglScene" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>701</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="src">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Source Image Directory :</string>
    </property>
   </widget>
   <widget class="QLabel" name="des">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Destination Directory :</string>
    </property>
   </widget>
   <widget class="QLabel" name="processLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtSrcAddress">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDesAddress">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="fileDialogBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="saveDialogBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Quit</string>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="dirOpenFile">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Directly open PLY file:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtOpenFileAddress">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="fileOpenDialogBtn">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileOpenBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="trim">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trim Script File:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtTrimScriptFile">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>210</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="fileTrimScriptBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnUp">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDown">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnLeft">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>590</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRight">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>590</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
